addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/InputWithDabBandControl.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol;
+
+import com.google.common.collect.Sets;
+import org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants;
+
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * The DAB Band control protocol interface.
+ *
+ * @author Tomasz Maruszak - [yamaha] Tuner band selection and preset feature for dual band models (RX-S601D)
+ */
+public interface InputWithDabBandControl extends IStateUpdateable {
+    /**
+     * List all inputs that are compatible with this kind of control
+     */
+    Set<String> supportedInputs = Sets.newHashSet(YamahaReceiverBindingConstants.INPUT_TUNER);
```

Comment:
@martinvw Java in most cases is inefficient enough. Just to get rid of a well-known dependency (I would say Guava and Apache Utils are well-known, accepted dependencies), should we really first create an implicit array, that is transformed to a list and then inserted into a hash?